packages/rest/src/rest.application.ts

```diff
@@ -68,6 +76,20 @@ export class RestApplication extends Application implements HttpServerLike {
   constructor(config: ApplicationConfig = {}) {
     super(config);
     this.component(RestComponent);
+    if (config.enableExplorer) {
+      const name = ExplorerComponent.name;
+      const componentKey = `components.${name}`;
+      this.bind(componentKey)
+        .toClass(ExplorerComponent)
+        .inScope(BindingScope.SINGLETON)
+        .tag('component');
+      // Assuming components can be synchronously instantiated
+      const explorerInstance = this.getSync<Component>(componentKey);
+      mountComponent(this, explorerInstance);
+      console.log(explorerInstance.staticAssets);
+      const explorer = explorerInstance.staticAssets![0];
+      this.static(explorer.path, explorer.rootDir, explorer.options);
```

Comment:
This block needs to be split into two parts:

`RestApplication` class should override `component` method provided by `Application`, iterate over `component.staticAssets` and call `this.static` to register them.

The scaffolded project-specific application class should register the component in the constructor, e.g.

```ts
class TodoApplication extends RepositoryMixin(BootMixin(RestApplication)) {
  constructor () {
    // ...
    this.component(ExplorerComponent);
  }
}
```